internal/bminventory/inventory.go

```diff
@@ -1084,7 +1085,7 @@ func (b *bareMetalInventory) createAndUploadNewImage(ctx context.Context, log lo
 			return common.NewApiError(http.StatusInternalServerError, err)
 		}
 	} else {
-		baseISOName, err := b.objectHandler.GetBaseIsoObject(infraEnv.OpenshiftVersion)
+		baseISOName, err := b.objectHandler.GetBaseIsoObject(infraEnv.OpenshiftVersion, versions.DefaultCPUArchitecture)
```

Comment:
Don't we save the arch as part of the cluster?
If it is supposed to come on a following PR - then add a TODO with the ticket key every place needs to be changed.

Reply:
Right, it'll be part of an upcoming PR. will add a TODO.